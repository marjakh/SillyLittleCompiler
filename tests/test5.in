write(1 + 2 - 3 + 4);
write(0 + 1 + 2 + 3);
write(0 - 1 - 2 - 3);
write(0 - (1 + 2) - 3);
write(3 * 2 / 3 + 4);
write(4 * 5 - 6 * 7);
write(4 / 8 + 1);
