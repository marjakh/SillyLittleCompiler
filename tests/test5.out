4
6
-6
-6
6
-22
1
